scripts/tasks/ts.ts

```diff
@@ -22,6 +22,15 @@ function prepareTsTaskConfig(options: TscTaskOptions) {
     options.sourceMap = true;
   }
 
+  const { isUsingV8pathAliases, tsConfigFileV8 } = getTsPathAliasesConfigV8();
+
+  if (isUsingV8pathAliases) {
+    logger.info(`📣 TSC: V8 package is using TS path aliases. Overriding tsconfig settings.`);
+    options.baseUrl = '.';
```

Comment:
turning path aliases off for actual production compilation to not break existing flows